Library/Homebrew/dev-cmd/bump.rb

```diff
@@ -0,0 +1,108 @@
+# frozen_string_literal: true
+
+require "cli/parser"
+
+module Homebrew
+  module_function
+
+  def bump_args
+    Homebrew::CLI::Parser.new do
+      usage_banner <<~EOS
+        `bump`
+
+        Display out-of-date brew formulae, the latest version available, and whether a pull request has been opened.
+      EOS
+      switch :verbose
+      switch :debug
+    end
+  end
+
+  def bump
+    bump_args.parse
+
+    outdated_repology_packages = Repology.parse_api_response
+    outdated_packages = validate_and_format_packages(outdated_repology_packages)
+
+    display(outdated_packages)
+  end
+
+  def validate_and_format_packages(outdated_repology_packages)
+    ohai "Verifying outdated repology packages as Homebrew formulae"
+
+    packages = {}
+    outdated_repology_packages.each do |_name, repositories|
+      # identify homebrew repo
+      repology_homebrew_repo = repositories.find do |repo|
+        repo["repo"] == "homebrew"
+      end
+
+      next if repology_homebrew_repo.blank?
+
+      latest_version = repositories.find { |repo| repo["status"] == "newest" }["version"]
+
+      packages[repology_homebrew_repo["srcname"]] = format_package(repology_homebrew_repo["srcname"], latest_version)
```

Comment:
maybe we could `srcname = repology_homebrew_repo["srcname"]` to avoid accessing the hash twice.